website/pages/docs/job-specification/group.mdx

```diff
@@ -129,12 +143,49 @@ group "example" {
 }
 ```
 
+### Stop After Client Disconnect
+
+This example shows how `stop_after_client_disconnect` interacts with
+other stanzas. For the `first` group, after the default 10 second
+[`heartbeat_grace`] window expires and 90 more seconds passes, the
+server will reschedule the allocation. The client will wait 90 seconds
+before sending a stop signal (`SIGTERM`) to the `first-task`
+task. After 15 more seconds because of the task's `kill_timeout`, the
+client will send `SIGKILL`. The `second` group does not have
+`stop_after_client_disconnect`, so the server will reschedule the
+allocation after the 10 second [`heartbeat_grace`] expires. It will
+not be stopped on the client, regardless of how long the client is out
+of touch.
+
+Note that if the server's clock is not closely synchronized with the
+client's clock, the server may reschedule before the client has
+stopped the allocation. Operators should ensure that clock drift
+between clients and servers is as small as possible.
+
```

Comment:
Note also that a group using this feature will be stopped on the client if the Nomad server cluster fails, since the client will be unable to contact any server in that case. Groups opting in to this feature are therefore exposed to an additional runtime dependency and potential point of failure.

Reply:
Done!